### ISSUE TITLE
NVDA is not announcing the label name along with the edit box name.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#14897]
### Steps to reproduce:
1.Click on the link https://jsfiddle.net/aqfhwvxL/1/
2.Launch the screen reader NVDA
3. Press ESC button to exit the Editor and press "B" button to navigate "Place Holder" button.
4. Press Down key to navigate to “event image” edit box, and check If the NVDA is announcing the label name along with the edit box name.
### Actual behavior:
When navigating to the “Event image” edit box screen reader is not announcing the label name along with the edit box name

### Expected behavior:
When navigating to the “Event image” edit box screen reader should be announcing the label name along with the edit box name

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
NVDA version 2022.4
#### Windows version:
Windows 11 Enterprise 22H2 22621.1265
#### Name and version of other software in use when reproducing the issue:
Google chrome Version 110.0.5481.178 (Official Build) (64-bit).
#### Does the issue still occur after restarting your computer?
Yes
[3054463_Chrome+NVDA.webm](https://user-images.githubusercontent.com/93735775/235851304-a322e92a-ef30-4b04-a633-2083761934a1.webm)

**Comment by Adriani90** [comment]
Might be related to #14840. Here a role="presentation" is used of the span and for the div which wrap the label. Not sure if this causes it though.

Is this also occuring in firefox?
cc: @jcsteh

**Comment by msftedad** [comment]
@Adriani90 This issue is also occurring in Firefox.

**Comment by seanbudd** [comment]
It seems that the steps here do not match the sample.
Can you explain what you are referring to with the "edit box" control? There are no edit boxes in the sample.

Can you also please ensure the HTML sample is **minimal**.

Please reproduce this issue and provide a [log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) of the behaviour. Ensure your log level is set to debug in general preferences.

**Comment by msftedad** [comment]
@seanbudd Sure, Will update once we get back the details from Dev team.

**Comment by msftedad** [comment]
@seanbudd 
here is the updated sample app https://codepen.io/Roopesh-T/pen/ExdGwdL with the Minimum steps provided by PO

1. Click on the link https://codepen.io/Roopesh-T/pen/ExdGwdL
2. Run NVDA
3. Tab to “Placeholder" button and press one more tab to move to "---" 
4. check If the NVDA is announcing the label name "Event image".

"the main issue here is that the label "---" has an aria label that is never read and didn't figure out why"

**Comment by msftedad** [comment]
Hi any update on this ticket ???

**Comment by seanbudd** [comment]
I can reproduce this in focus mode but not browse mode

**Comment by jcsteh** [comment]
You mention an edit box, but there is no edit box that I can see, only a focusable div with no semantics. Even that div isn't labelled by the Event image label.

**Comment by seanbudd** [comment]
@jcsteh the updated steps are in https://github.com/nvaccess/nvda/issues/14897#issuecomment-1556978231

@msftedad - please update the actual issue with the correct steps to reproduce

**Comment by jcsteh** [comment]
Even in that example, the focusable div after the button has an aria-label of "" and a title, but not aria-labelledby. It is not labelled by the "Event image" label.

**Comment by jcsteh** [comment]
My comments in https://github.com/nvaccess/nvda/issues/14897#issuecomment-1644876916 still hold. That field isn't an edit box and the label isn't associated with the focusable element. Even if it were, the label is visible, so it is read where it occurs; i.e. on the line above.

**Comment by msftedad** [comment]
Hi @jcsteh any update on this issue.

**Comment by jcsteh** [comment]
Nothing new because the test case is still incorrect and doesn't match your description.

> That field isn't an edit box and the label isn't associated with the focusable element. Even if it were, the label is visible, so it is read where it occurs; i.e. on the line above.

To be clear, this test case is broken. This is not an NVDA problem.

**Comment by jcsteh** [comment]
Maybe it *looks like* an edit box, but it isn't semantically according to the code. Either way, it's broken.
